Gruntfile.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+// Because you can't specify a directory as target for grunt-browserify we have to generate the file configuration at runtime
```

Comment:
Sorry about all the Hound comments earlier. I removed them. This line is indeed a bit long, though. ;)